### PR TITLE
sql/migrate: default Scanner implementation

### PR DESCRIPTION
**Pull request by masseelch**

### CODE REVIEW

**Review by hedwigz — APPROVED** [review]
:100:

**Comment by a8m** [comment]
Where's the default implementation?

**Comment by masseelch** [comment]
> Where's the default implementation?

It somehow ended up in the PR before. I am sorry :-(

File: sql/migrate/migrate.go [hunk 1]
```diff
@@ -312,6 +311,8 @@ func (p *Planner) WritePlan(plan *Plan) error {
 	return nil
 }
 
+var ErrInSync = errors.New("sql/migrate: execute: nothing to do")
```
Comment (a8m) [inline sql/migrate/migrate.go:314]: `ErrNoFiles`/`ErrNoRevisions` or something else

**Review by a8m — APPROVED** [review]

**Pull request merged**